libs/order-list/src/setup-tests.ts

```diff
@@ -1 +1,2 @@
 import '@testing-library/jest-dom';
+import 'jest-canvas-mock';
```

Comment:
question (blocking): why is this needed?

Reply:
can be removed, updated in last commit.